### PR TITLE
Updating SignToolData to include the produced nuget packages

### PR DESCRIPTION
**Pull request by tannergooding**

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @livarcocc

File: build/SignToolData.json [hunk 1]
```diff
@@ -4,15 +4,133 @@
...
+        },
+        {
+            "certificate": null,
+            "strongName": null,
+            "values": [
+                "packages/*.nupkg"
             ]
         }
+    ],
+    "exclude": [
```
Comment (livarcocc) [inline build/SignToolData.json]: Do you really need this exclude section if you are being explicit on the dlls to sign above?

Comment (tannergooding) [reply build/SignToolData.json]: Yes. The SignTool expects all binaries in containers (nuget, vsix, etc) to be explicitly signed or explicitly excluded.

Comment (tannergooding) [reply build/SignToolData.json]: CC. @jaredpar

**Review by livarcocc — APPROVED** [review]

**Pull request merged**